paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/ControllerConfiguration.java

```diff
@@ -148,26 +213,15 @@ public EventHandlerController(final EventReceiverRegistry registry, final Client
             return deferredResult;
         }
 
-        private Predicate<EventTypeConsumer> filerConsumer(@Nonnull final String eventName,
```

Comment:
This method was moved to `ReplayHandler` but is only used in `ControllerConfiguration` (here).